main.cpp

```diff
@@ -1,3 +1,8 @@
+#include <iostream>
+
+#include <iostream>
+#include <string.h>
```

Comment:
included header string.h is not used directly [misc-include-cleaner,-warnings-as-errors]